### ISSUE TITLE
wlfreerdp: `/app:"||..."` doesn't work

### ISSUE THREAD

**Issue by tchernomax** [FreeRDP/FreeRDP#6573]
**Describe the bug**

I use swaywm (a wayland environment).

When I run:
```wlfreerdp /u:... /d:.... /p:"…" /app:"||VpxClient" /v:...```
I see a new window appearing, with connections information, the connection goes on without error.
(a window a bit like this : https://support.eapps.com/eapps_images/win_images/rdp12_img/winxp_remote_login.png ) 

Then when the connection is finished the window stay black. No app seems to run.

When I run the exact same command with xfreerdp:
```xfreerdp /u:... /d:.... /p:"…" /app:"||VpxClient" /v:...```
I get the same connection information, and then I get the "VpxClient" app.
(But I have glitch and pointer location problem with xfreerdp so I can't use it)

When I kill xfreerdp and then relaunch wlfreerdp:
```wlfreerdp /u:... /d:.... /p:"…" /app:"||VpxClient" /v:...```
Then I get the "VpxClient" app window.

But when I try to run another application:
```wlfreerdp /u:... /d:.... /p:"…" /app:"||chrome" /v:...```
I still get the "VpxClient" app.

If I clic the exit button on "VpxClient", and try to relaunch it:
```wlfreerdp /u:... /d:.... /p:"…" /app:"||VpxClient" /v:...```
I get a black screen.

Note: I always get the connections/login screen, without problem.

It seems wlfreerdp doesn't apply the "/app:..." directive.
Currently I can bypass this issue by launching the app via xfreerdp, killing xfreerpd and launching wlfreerdp.

**To Reproduce**

```wlfreerdp /u:... /d:.... /p:"…" /app:"||myapp" /v:...```

When myapp isn't already running on the remote server

-> after authentication, you get a black window  
-> "myapp" isn't launch

**Expected behavior**

"myapp" should be launch on the remote server.

**Application details**
* FreeRDP version: `This is FreeRDP version 2.2.0 (n/a)`
* Output of `wlfreerdp /buildconfig`: 
```
This is FreeRDP version 2.2.0 (n/a)
Build configuration: BUILD_TESTING=OFF BUILTIN_CHANNELS=ON HAVE_AIO_H=1 HAVE_EXECINFO_H=1 HAVE_FCNTL_H=1 HAVE_GETLOGIN_R=1 HAVE_INTTYPES_H=1 HAVE_JOURNALD_H=TRUE HAVE_MATH_C99_LONG_DOUBLE=1 HAVE_POLL_H=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK=ON HAVE_PTHREAD_MUTEX_TIMEDLOCK_LIB=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK_SYMBOL= HAVE_SYSLOG_H=1 HAVE_SYS_EVENTFD_H=1 HAVE_SYS_FILIO_H= HAVE_SYS_MODEM_H= HAVE_SYS_SELECT_H=1 HAVE_SYS_SOCKIO_H= HAVE_SYS_STRTIO_H= HAVE_SYS_TIMERFD_H=1 HAVE_TM_GMTOFF=1 HAVE_UNISTD_H=1 HAVE_XI_TOUCH_CLASS=1 WITH_ALSA=ON WITH_CAIRO=OFF WITH_CCACHE=ON WITH_CHANNELS=ON WITH_CLANG_FORMAT=ON WITH_CLIENT=ON WITH_CLIENT_AVAILABLE=1 WITH_CLIENT_CHANNELS=ON WITH_CLIENT_CHANNELS_AVAILABLE=1 WITH_CLIENT_COMMON=ON WITH_CLIENT_INTERFACE=OFF WITH_CUPS=ON WITH_DEBUG_ALL=OFF WITH_DEBUG_CAPABILITIES=OFF WITH_DEBUG_CERTIFICATE=OFF WITH_DEBUG_CHANNELS=OFF WITH_DEBUG_CLIPRDR=OFF WITH_DEBUG_DVC=OFF WITH_DEBUG_KBD=OFF WITH_DEBUG_LICENSE=OFF WITH_DEBUG_MUTEX=OFF WITH_DEBUG_NEGO=OFF WITH_DEBUG_NLA=OFF WITH_DEBUG_NTLM=OFF WITH_DEBUG_RAIL=OFF WITH_DEBUG_RDP=OFF WITH_DEBUG_RDPDR=OFF WITH_DEBUG_RDPEI=OFF WITH_DEBUG_RDPGFX=OFF WITH_DEBUG_REDIR=OFF WITH_DEBUG_RFX=OFF WITH_DEBUG_RINGBUFFER=OFF WITH_DEBUG_SCARD=OFF WITH_DEBUG_SND=OFF WITH_DEBUG_SVC=OFF WITH_DEBUG_SYMBOLS=OFF WITH_DEBUG_THREADS=OFF WITH_DEBUG_TIMEZONE=OFF WITH_DEBUG_TRANSPORT=OFF WITH_DEBUG_TSG=OFF WITH_DEBUG_TSMF=OFF WITH_DEBUG_TSMF=OFF WITH_DEBUG_TSMF_AVAILABLE=0 WITH_DEBUG_URBDRC=OFF WITH_DEBUG_WND=OFF WITH_DEBUG_X11=OFF WITH_DEBUG_X11_CLIPRDR=OFF WITH_DEBUG_X11_LOCAL_MOVESIZE=OFF WITH_DEBUG_XV=OFF WITH_DSP_EXPERIMENTAL=OFF WITH_DSP_FFMPEG=ON WITH_EVENTFD_READ_WRITE=1 WITH_FAAC=OFF WITH_FAAD2=OFF WITH_FFMPEG=TRUE WITH_FFMPEG=TRUE WITH_GFX_H264=ON WITH_GPROF=OFF WITH_GSM=OFF WITH_GSSAPI=OFF WITH_ICU=OFF WITH_IPP=OFF WITH_JPEG=ON WITH_LAME=OFF WITH_LIBRARY_VERSIONING=ON WITH_LIBSYSTEMD=ON WITH_MACAUDIO=OFF WITH_MACAUDIO=OFF WITH_MACAUDIO_AVAILABLE=0 WITH_MANPAGES=ON WITH_MBEDTLS=OFF WITH_OPENCL=OFF WITH_OPENH264=OFF WITH_OPENSLES=OFF WITH_OPENSSL=ON WITH_OSS=ON WITH_PAM=ON WITH_PCSC=ON WITH_PROFILER=OFF WITH_PROXY=ON WITH_PROXY_MODULES=OFF WITH_PULSE=ON WITH_SAMPLE=OFF WITH_SANITIZE_ADDRESS=OFF WITH_SANITIZE_ADDRESS_AVAILABLE=1 WITH_SANITIZE_MEMORY=OFF WITH_SANITIZE_MEMORY_AVAILABLE=1 WITH_SANITIZE_THREAD=OFF WITH_SANITIZE_THREAD_AVAILABLE=1 WITH_SERVER=ON WITH_SERVER_CHANNELS=ON WITH_SERVER_INTERFACE=ON WITH_SHADOW=ON WITH_SMARTCARD_INSPECT=OFF WITH_SOXR=OFF WITH_SSE2=ON WITH_SWSCALE=ON WITH_THIRD_PARTY=OFF WITH_VAAPI=ON WITH_VALGRIND_MEMCHECK=OFF WITH_VALGRIND_MEMCHECK_AVAILABLE=1 WITH_WAYLAND=ON WITH_WINPR_TOOLS=ON WITH_X11=ON WITH_X264=OFF WITH_XCURSOR=ON WITH_XDAMAGE=ON WITH_XEXT=ON WITH_XFIXES=ON WITH_XI=ON WITH_XINERAMA=ON WITH_XKBFILE=ON WITH_XRANDR=ON WITH_XRENDER=ON WITH_XSHM=ON WITH_XTEST=ON WITH_XV=ON WITH_ZLIB=ON
Build type:          None
CFLAGS:              -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fPIC -Wall -Wno-unused-result -Wno-unused-but-set-variable -Wno-deprecated-declarations -fvisibility=hidden -Wimplicit-function-declaration -Wredundant-decls -g -fno-omit-frame-pointer -DWINPR_DLL
Compiler:            GNU, 10.2.0
Target architecture: x64
```
* OS version connecting to (server side): no clue (it's run by our IT)
* If available the log output from a run with `/log-level:trace 2>&1 | tee log.txt`: [log.txt](https://github.com/FreeRDP/FreeRDP/files/5530573/log.txt)

**Environment :**
 - OS: Linux
 - Version/Distribution: Archlinux uptodate
 - Architecture: x86_64

**Comment by akallabeth** [comment]
yes, wayland client does not support rail.

**Comment by akallabeth** [comment]
will not be implemented, replaced by sdl-freerdp.

**Comment by bmullan** [comment]
@akallabeth 

system - Ubuntu 24.04 LTS
**freerdp3 works fine running a remoteapp with x11** but fails with just a frozen black screen in a wayland session.
so in Ubuntu 24.04 there is still the option at the login menu to choose x11 or wayland
for now that's find

However, with Ubuntu 26.04 LTS released using Gnome 50 and now xorg (x11) no longer supported & no choice at the login menu either.

wlfreerdp3 is deprecated and mentions using sdl-freerdp but 
I compiled sdl-freerdp3 from source and it works for a full Windows Desktop but not for remoteapp execution as a separate window does pop up but it appears frozen & also looks like it was not running the remoteapp chosen

Note:  I also tried xfreerdp3 with xwayland installed and it won't run remote apps (/app:) either.

What options are there with freerdp and wayland on Ubuntu 24.04 or even more so, with 26.04?
